scripts/index.js

```diff
@@ -0,0 +1,117 @@
+const state = {
+    temp: 82,
+    sky: "Sunny",
+    city: "Emerald City"
+};
+
+const skys = [`☁️ ☁️ ☁️ ☀️ ☁️ ☁️`,
+    `☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️`,
+    `🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧`,
+    `🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨`
+]
+
+const landscapes = [
+    `🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂`,
+    `🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷`,
+    `🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃`,
+    `🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲`
+]
+
+const incrementTemp = () => {
+    let temp = document.getElementById("temp");
+    state.temp += 1;
+    temp.innerHTML = state.temp;
+    const landscapeIdx = getLandscapeIdx()
+
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const decrementTemp = () => {
+    const temp = document.getElementById("temp");
+    state.temp -= 1;
+    temp.innerHTML = state.temp;
+
+    const landscapeIdx = getLandscapeIdx()
+    const landscapeImg = document.getElementById("landscapeImg");
+    landscapeImg.innerHTML = landscapes[landscapeIdx]
+}
+
+const getLandscapeIdx = () => {
+    if (state.temp >= 80) {
+        return 0
+    } else if (state.temp >= 70) {
+        return 1
+    } else if (state.temp >= 60) {
+        return 2
+    } else {
+        return 3
+    }
+}
+
+const getSkyIdx = () => {
+    switch (state.sky) {
+        case 'Sunny':
+            return 0
+        case 'Cloudy':
+            return 1
+        case 'Rainy':
+            return 2
+        case 'Snowy':
+            return 3
+        default:
+            break
+    }
+}
+
+function setCity() {
+    const newCity = document.getElementById("cityInput").value;
+    state.city = newCity;
+    document.getElementById("city").innerHTML = `for ✨${newCity}✨`;
+}
+
+function resetCity() {
+    state.city = "Emerald City";
```

Comment:
I'd like to see the "Emerald City" default string stored in a const once, then used here and to initialize the default state. It's usually a good idea to minimize the number of bare strings in our code, since those have a tendency to get out of sync with one another over time and are typo-prone.